tools/java/org/unicode/cldr/api/CldrValue.java

```diff
@@ -70,7 +71,13 @@ static CldrValue create(String value, Map<AttributeKey, String> valueAttributes,
     private final int hashCode;
 
     private CldrValue(String value, Map<AttributeKey, String> attributes, CldrPath path) {
-        // Empty value strings are permitted (but null is not).
+        // Since early 2019 there's been the possibility of getting the inheritance marker as
+        // a value for a path. This indicates that the value does NOT actually exist for a
+        // locale and would be inherited. However everything that creates a CldrValue instance
+        // is expected to deal with this and we should never see inheritance markers here.
+        // Note: This also serves as a null check for values.
+        checkArgument(!value.equals(CldrUtility.INHERITANCE_MARKER),
+            "unexpected inheritance marker '%s' for path: %s", value, path);
```

Comment:
If you are running on trunk, which people may be doing during development, will people see a slew of console messages?

Reply:
They certainly shouldn't. This happens as a last resort test after they should have all been caught and dealt with. As in it signifies a (serious) code error in the API rather than a data issue.